Resources/Prototypes/Entities/Objects/Consumable/Food/Baked/pizza.yml

```diff
@@ -547,3 +547,73 @@
         - ReagentId: Vitamin
           Quantity: 1
 # Tastes like stale crust, rancid cheese, mushroom.
+
+- type: entity
+  name: spicy rock pizza
+  parent: FoodPizzaBase
+  id: FoodPizzaUranium
+  description: Spicy pizza covered in peppers and uranium.
+  components:
+  - type: FlavorProfile
+    flavors:
+    - spicy
+    - cheesy
+    - oily
+    - bread
+  - type: Sprite
+    layers:
+    - state: uranium-pizza
+  - type: SliceableFood
+    slice: FoodPizzaUraniumSlice
+  - type: Tag
+    tags:
+    - Meat
+    - Pizza
+  - type: PointLight
+    enabled: true
+    radius: 2
+
+  - type: SolutionContainerManager
+    solutions:
+      food:
+        maxVol: 5
+        reagents:
+        - ReagentId: Nutriment
+          Quantity: 100
+        - ReagentId: Radium
+          Quantity: 80
```

Comment:
35-40 total reagents is the standard, this is far too high.

Reply:
I went ahead and fixed this and also split the reagents between nutriment, radium and uranium to better relate to the material thats used to craft it, keep quantities consistent between cut and uncut pizza and Bringing it down to 40 reagent total for a full pizza.